frontend/src/views/Crops/CropsModal.tsx

```diff
@@ -773,7 +785,11 @@ function CropsModal({
                       isRequired
                       name="whereWillPruningsGo"
                       items={whereWillPruningsGo.map((ele) => ({ id: ele.id, label: ele.label }))}
-                      selectedKey={formData.whereWillPruningsGo || 0}
+                      selectedKey={
+                        whereWillPruningsGo.find(
+                          (option) => option.label === formData.whereWillPruningsGo,
```

Comment:
Technically you could do the same thing here, since id for Select elements can be a string.